.azure-pipelines-templates/model_checking.yml

```
@@ -27,13 +27,39 @@ jobs:
       - script: ./tlc.sh -workers auto -config consensus/MCccfraftAtomicReconfig.cfg consensus/MCccfraft.tla -dumpTrace tla MCccfraftAtomicReconfig.trace.tla -dumpTrace json MCccfraftAtomicReconfig.json
         workingDirectory: tla
         displayName: MCccfraftAtomicReconfig.cfg
+        condition: ne(variables['Build.CronSchedule.DisplayName'], 'Daily build')
 
       - script: ./tlc.sh -workers auto -config consensus/MCccfraftWithReconfig.cfg consensus/MCccfraft.tla -dumpTrace tla MCccfraftWithReconfig.trace.tla -dumpTrace json MCccfraftWithReconfig.json
         workingDirectory: tla
         displayName: MCccfraftWithReconfig.cfg
+        condition: ne(variables['Build.CronSchedule.DisplayName'], 'Daily build')
+
+      - script: ./tlc.sh -workers auto -config consensus/MCccfraftAtomicReconfig.cfg consensus/MCccfraft.tla -dumpTrace tla MCccfraftAtomicReconfig.trace.tla -dumpTrace json MCccfraftAtomicReconfig.json -coverage 1 -nowarning
+        workingDirectory: tla
+        displayName: MCccfraftAtomicReconfig.cfg
+        condition: eq(variables['Build.CronSchedule.DisplayName'], 'Daily build')
+
+      - script: ./tlc.sh -workers auto -config consensus/MCccfraftWithReconfig.cfg consensus/MCccfraft.tla -dumpTrace tla MCccfraftWithReconfig.trace.tla -dumpTrace json MCccfraftWithReconfig.json -coverage 1 -nowarning
+        workingDirectory: tla
+        displayName: MCccfraftWithReconfig.cfg
+        condition: eq(variables['Build.CronSchedule.DisplayName'], 'Daily build')
+
+      - script: |
+          set -ex
+          python3 -mvenv env
+          source env/bin/activate
+          pip install -r requirements.txt
+          ls -tr MCccfraftAtomicReconfig_coverage_*.json | xargs cat | python3 actions.py MCccfraftAtomicReconfig.html
+          rm MCccfraftAtomicReconfig_coverage_*.json
+          ls -tr MCccfraftWithReconfig_coverage_*.json | xargs cat | python3 actions.py MCccfraftWithReconfig.html
+          rm MCccfraftWithReconfig_coverage_*.json
+          deactivate
+          rm -rf env
+        workingDirectory: tla
+        condition: eq(variables['Build.CronSchedule.DisplayName'], 'Daily build')
 
       - task: PublishPipelineArtifact@1
         inputs:
           artifactName: "Model Checking Traces"
           targetPath: tla
-        condition: or(failed(), canceled())
+          condition: or(or(failed(), canceled()), eq(variables['Build.CronSchedule.DisplayName'], 'Daily build'))
```

```suggestion
        condition: or(or(failed(), canceled()), eq(variables['Build.CronSchedule.DisplayName'], 'Daily build'))
```

I think this is now indented too far, so the condition isn't applied?